indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -521,6 +521,31 @@ def _process(self):
                                                     user_data=self._get_user_data(), management=True)
 
 
+class RHRegistrationCreateMultiple(RHManageRegFormBase):
+    """Create multiple registrations for Indico users (management area)"""
+
+    def _register_user_personal_data_only(self, user, notify):
+        data = {pdt.name: getattr(user, pdt.name, None) for pdt in PersonalDataType}
+        with db.session.no_autoflush:
+            create_registration(self.regform, data, management=True, notify_user=notify)
+
+    def _process(self):
+        form = CreateMultipleRegistrationsForm(regform=self.regform, open_add_user_dialog=(request.method == 'GET'),
+                                               notify_users=session.get('registration_notify_user_default', True))
+
+        if form.validate_on_submit():
+            session['registration_notify_user_default'] = form.notify_users.data
+            for user in form.user_principals.data:
+                self._register_user_personal_data_only(user, form.notify_users.data)
+            reg_list_config = _get_reg_list_config(regform=self.regform)
```

Comment:
is this code block (and the lines below) repeated, e.g. in the other RH to add a registration? if yes it'd be useful to move it into a function. e.g. `jsonify_registration_list(regform)`